pkg/protostore/protostore.go

```diff
@@ -20,6 +20,8 @@
 // around [pkg.datastore.DataStore] interface and is used specifically to persist Protobuf messages.
 // Just as with [pkg.datastore.DataStore], Protobuf messages can be persisted with revisioning and
 // multi-tenancy support along with `CreatedAt` and `UpdatedAt` timestamps.
+// Tombstone Delete or soft deletes are supported with `DeletedAt` struct field.
+// Use Delete to do remove any records that are soft deleted but still in database
```

Comment:
You should change "Use Delete to **do** remove any records that are soft deleted but still in database" to "Use Delete to remove any records that are soft deleted but still in database"

Reply:
@krishnamiriyala, I think you forgot to push/amend a commit